src/modules/Builder/features/PerformanceTasks/Flanker/RoundSettings/RoundOptions/RoundOptions.tsx

```diff
@@ -1,26 +1,27 @@
+import { Checkbox, FormControlLabel } from '@mui/material';
 import { ChangeEvent } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Checkbox, FormControlLabel } from '@mui/material';
```

Comment:
Minor nit: I think we (mostly) follow convention to keep `react` import at the top. Feel free to ignore.

Reply:
Updating :)